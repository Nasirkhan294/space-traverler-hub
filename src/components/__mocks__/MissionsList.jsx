import React from 'react';

const MissionsList = () => (
  <div>
    <table>
      <tr>
        <th>Mission</th>
        <th>Description</th>
        <th>Status</th>
        <th>Join</th>
      </tr>
    </table>
  </div>
);

export default MissionsList;
