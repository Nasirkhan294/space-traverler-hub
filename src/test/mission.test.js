import React from 'react';
import renderer from 'react-test-renderer';
import MissionsList from '../components/__mocks__/MissionsList';

describe('MissionsList', () => {
  it('renders correctly', () => {
    const component = renderer.create(<MissionsList />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
