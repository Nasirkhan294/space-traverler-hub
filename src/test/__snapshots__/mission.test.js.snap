// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MissionsList renders correctly 1`] = `
<div>
  <table>
    <tr>
      <th>
        Mission
      </th>
      <th>
        Description
      </th>
      <th>
        Status
      </th>
      <th>
        Join
      </th>
    </tr>
  </table>
</div>
`;
